zipline/src/hostTest/kotlin/app/cash/zipline/ZiplineSecurityTest.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2024 Block, Inc.
```

Reply:
Other files in this directory are attributed to Square. Follow up to clean these up and change to Block? (some are 2022 when block existed)